doc/integrator/database.rst

```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
 and ``production_static`` for the production environment.
 
 For the data of the ``main`` schema, we recommend to have one schema for each version of the application.
-The following example shows how an update can be performed from a version ``2017`` to a version ``2018``:
+The following example shows how an update can be performed from a version ``2019`` to a version ``2020``:
+
+========================  =============  =============
+                           Integration    Production
+                           schema name    schema name
+========================  =============  =============
+Initial state              main_2019      main_2019
+Sart a new version         main_2020      main_2019
+Do the changes             main_2020      main_2019
+Publish the new version    main_2020      main_2020
+========================  =============  =============
+
+Initial state
+~~~~~~~~~~~~~
 
-* Starting point is that the current version is the same on integration and production => ``main_2017``
-* When starting the changes, such as an application change and/or administration settings,
-  create a new schema ``main_2018`` and use it on integration. Now, integration uses ``main_2018``,
-  while production still uses ``main_2017``.
-* Do the changes or the upgrade.
-* Publish the new version on production: now, integration and production both use ``main_2018``
-* The schema ``main_2017`` still exists, so if needed, the production can be rolled back to this content.
+Starting point is that the current version is the same on integration and production => ``main_2019``.
+
+Prepare the project
+~~~~~~~~~~~~~~~~~~~
 
 To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
 should be managed in your makefiles:
 
-* the ``PGSCHEMA`` variable should be set in the ``Makefile``.
-* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile for the production e.-g. ``production.mk``.
+* the ``PGSCHEMA`` variable should be set in the ``Makefile``, she will be set to ``main_2019``.
+* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile
+  for the production e.-g. ``production.mk``, she will be set to ``integration_static`` in the
+  Mankefile, and to ``production_static`` in the ``production.mk`` file.
```

Comment:
We now mention two times "in the production.mk" file in the same sentence, we should simplify this sentence